packages/components/src/internal/components/settings/NameIdSettings.tsx

```diff
@@ -47,18 +47,19 @@ interface NameIdSettingsProps extends InjectedRouteLeaveProps {
 interface State {
     allowUserSpecifiedNames: boolean;
     api?: ComponentsAPIWrapper;
+    confirm: string[];
```

Comment:
`confirm` seems like a weird name for this. It is actually a list of `prefixIneligibleSampleTypeNames`.